index.html

```diff
@@ -33,6 +33,293 @@
 </head>
 
 <body>
+  <section id="abstract">
+    <h2>Abstract</h2>
+    <p>
+      This document defines a new API {{MediaDevices/getAllScreenMedia}}
+      for capturing multiple
+      <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+      following one user gesture.
+      It is an extension to the Screen Capture API [[screen-capture]].
+    </p>
+  </section>
+  <section id="sotd">
+    <p>
+      This document is a draft. It is subject to major changes and, while early
+      experimentations are encouraged, it is therefore not intended for implementation.
+    </p>
+  </section>
+  <section class="informative" id="introduction">
+    <h2>
+      Introduction
+    </h2>
+    <p>
+      The Screen Capture API [[screen-capture]] enables the capturing of a single
+      <a data-cite="screen-capture#dfn-display-surface">display surface</a> in the
+      form of a video track.
+    </p>
+    <p>
+
+      Users currently cannot capture multiple
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      at once without a [=transient activation=]. This may be useful in certain
+      scenarios, e.g. for compliance or training purposes.
+      The only currently available alternative is through the Screen Capture API with
+      <a data-cite="screen-capture#dom-mediadevices-getdisplaymedia">getDisplayMedia</a>
+      by calling
+      <a data-cite="screen-capture#dom-mediadevices-getdisplaymedia">getDisplayMedia</a>
+      several times and asking the user to select the available
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>.
+      This requires multiple user gestures, repeated interaction with browser-level UX.
+      This adds unnecessary friction and results in degraded user experience and
+      interrupted workflows.
```

Comment:
It also provides no guarantee to the app that the user has indeed selected all screens. And in the case of a managed device running a pre-installed application, this means the app might very well not be able to guarantee that the user should even be allowed to interact with the device any further, e.g. if it creates a legal compliance risk.

Reply:
revised the paragraph.